src/clj/harja/palvelin/palvelut/yllapitokohteet/paikkauskohteet.clj

```diff
@@ -760,8 +783,8 @@
 
 (defn vastaanota-urem-excel [db req]
   (oikeudet/vaadi-kirjoitusoikeus oikeudet/urakat-paikkaukset-paikkauskohteetkustannukset
```

Comment:
Tässäkin sisennys poikkeaa sovitun mukaisesta, nyt näyttäisi olevan align parameters tms. päällä

Reply:
Hyvä huomio, korjailin näitä.